### DISCUSSION TITLE
Related research and recommendations for design and research

### DISCUSSION THREAD

**Discussion by calvin-lau-sig7** [alphagov/govuk-design-system#2910] [Exit this page]
For wider, service design insights, jobs to be done and recommendations if you are supporting people with a lived experience of domestic abuse, seek out Jenny Gibson’s extensive and insightful research with over 100 people with a lived experience of domestic abuse for the Department of Work and Pensions, ‘Designing Services that Keep People Safe’. 

Key insights we drew on for this project were:
- not everyone understands what domestic abuse means, including people who have or are experiencing it
- helpful digital guidance is often missed by those who need it
- many people feel no connection with the terminology
- headlines, links and content titled domestic abuse are ignored by a significant number of people who have experienced non violent forms of abuse
- people planning to leave may have snatched moments of time to access information online and they may be interrupted or have their activity checked afterwards.

Recommendations from this research were:
- recognise that some of your users could be at risk. Review your service as a person who has been compromised digitally.
- reconsider any signposting and content designed to support vulnerable users
- be conscious of time strapped users and consider ways to provide a quick exit or access in short bursts

Content design recommendations, taken from 'How to design content for people in crisis' by Miriam Vaswani, Content designer extraordinaire, UX Scotland 2023

Do:
- name the action
- be directive
- leave no room for misunderstanding
- learn what people need
- understand the risks in all reasonable contexts
- design for the environment

Consider, whether your design will work for someone who:
- shares a device
- can't charge their device
- can't avoid bright sunlight, rain or extreme cold
- can't find or afford a reliable internet connection
- is being watched by a coercive partner
- is in a dangerous environment
- needs to translate or interpret the language of your design

**Comment by melissaegg** [comment]
Hi @calvin-lau-sig7 this is great! Can you provide a link to Jenny Gibson’s research by chance? Is it public?

**Comment by melissaegg** [comment]
Also, Gov.UK provided me the art files in 2017 when I made a poster about designing for trauma back then, it's in a different repo: https://github.com/UKHomeOffice/posters/blob/master/README.md

**Comment by calvin-lau-sig7** [comment]
Hi Melisa, thanks for getting in touch! Our user researcher's shares lots of relevant research in the '[Useful resources](https://github.com/alphagov/govuk-design-system/discussions/2911)' discussion.

Unfortunately, I'm not sure we have Jenny Gibson's research to share, or whether it can be shared publicly. Unfortunately our user researcher, who'd know more about this, is not available in the short term. 

I'd suggest getting in touch with Jenny Gibson or user research at DWP.

**Reply by melissaegg** [reply]
Ok thanks, Calvin! I'm actually looking for that very specific research so I can track down Jenny. Already have [lots of resources related to trauma-informed design here. ](https://www.birdcallux.com/blog/resources-for-trauma-informed-ux-design-talk)

**Reply by calvin-lau-sig7** [reply]
Ahh thanks for sharing that list!
